openQA.spec

```diff
@@ -71,10 +71,8 @@
 %endif
 # The following line is generated from dependencies.yaml
 %define devel_no_selenium_requires %build_requires %qemu %test_requires curl perl(Devel::Cover) perl(Devel::Cover::Report::Codecov) perl(Perl::Tidy) postgresql-devel rsync sudo tar xorg-x11-fonts
-%ifarch x86_64 aarch64
 # The following line is generated from dependencies.yaml
 %define devel_requires %devel_no_selenium_requires chromedriver
```

Comment:
What about keeping the `%ifarch` but adding an

```
%else
%define devel_requires %devel_no_selenium_requires
```

Reply:
Forget my comment. Using `%ifarch` makes no sense when building a noarch package. We needed to make it an arch-specific package as soon if the make arch-specific things (which would actually be an option to consider).

Reply:
Actually I think the approach in this PR without any exclusion is correct as the dependency for a complete devel environment simply *is* including chromedriver. If we can not fulfill dependencies on the side of OBS then this is a problem that can be handled there accordingly, e.g. disable building/publishing of the subpackage.